src/arch/InputHandler/InputHandler.cpp

```diff
@@ -128,6 +128,8 @@ RString InputHandler::GetDeviceSpecificInputString( const DeviceInput &di )
 	if( di.device == InputDevice_Invalid )
 		return RString();
 
+    RString deviceString = InputDeviceToString(di.device) + " (ID: " + std::to_string(di.uniqueID) + ")";
```

Comment:
unused

Reply:
True, this ended up being reused below with a different variable name and I forgot to delete this.